src/backend/routes/metadata.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  var dtrFhirVersion = "4.0.1";
+  if (process.env.FHIR_VERSION) {
+    dtrFhirVersion = process.env.FHIR_VERSION;
+  }
+
+  var dtrRegisterURL = "http://localhost:3005/register";
+  if (process.env.DTR_REGISTER_URL) {
+    dtrRegisterURL = process.env.DTR_REGISTER_URL;
+  }
+
+  const capabilityStatement = {
```

Comment:
I might've missed this conversation, but I thought capability statements were for servers?  DTR would be the client, there really shouldn't be a need for a capability statement.  What's the use case for this?

Reply:
My understanding is that DTR is a SMART on FHIR server that provides client registration service and supports EHR Launch and SMART's Standalone Launch modes. And according to this SMART App Launch conformance http://hl7.org/fhir/smart-app-launch/conformance/ DTR needs to support both FHIR CapabilityStatement and Well-known URIs.

Reply:
I believe that conformance document refers to the server, or the EHR, not the client,  or DTR.

Reply:
I will update the capability statement and well-known URIs so they are appropriate for DTR as a client. 
A SMART app should have a capability statement indicating what launch modes it can have and what it can do when accessing a server/EHR.